File: easy solutions/0009_palindrome_number.py
# Given an integer x, return true if x is a palindrome, and false otherwise.

class Solution:
    def isPalindrome(self, x: int) -> bool:
        number_as_string = str(x)
        for number in range(1,len(number_as_string)):
            if (number_as_string[number] != number_as_string[-(number+1)]):
                return False

        return True 

File: c sharp/easy solutions/0009_palindrome.number.cs
/*

Given an integer x, return true if x is a palindrome, and false otherwise.

*/

public class Solution {
    public bool IsPalindrome(int x) {
        string numberToString = x.ToString();
        string reversedString = new string(numberToString.Reverse().ToArray());
        return numberToString == reversedString;
    }
}
